static-analysis/teammates-eslint-es6.yml

```diff
@@ -20,6 +20,15 @@ rules:
     # enforce a maximum line length
     max-len: ["error", 125]
 
+    # disallow continue statements
+    no-continue: "off"
+
+    # disallow mixes of different operators
+    no-mixed-operators: "off"
```

Comment:
`no-mixed-operators`: I haven't seen the full extent of this rule's capability yet, but it complains on each of the following:
```javascript
const largestValueInRange = min + (possibleValuesCount - 1) * step;

(Math.round((min + 2 * step) * 1000) / 1000).toString()}, ..., ${

(Math.round((max - 2 * step) * 1000) / 1000).toString()}, ${
```
which are not even that hard to understand.